app/assets/stylesheets/photos.scss

```diff
@@ -0,0 +1,26 @@
+// Place all the styles related to the photos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.photo_page {
+  text-align: center;
+}
+
+.comments {
+    display:inline-block;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space